### PR TITLE
BO - Product V2 - Priority management form is disabled

### LINKED ISSUE #28968: BO - Poduct V2 - Priority management is not working as expected
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

The Priority management is not working as expected
The specific priority was fixed here: https://github.com/PrestaShop/PrestaShop/pull/26938#pullrequestreview-923518880

### Expected behavior

https://github.com/PrestaShop/PrestaShop/issues/19674

### Steps to reproduce

1. Go to BO > Advanced Parameters > New & Experimental features page
2. Enable the New product page
3. Go to BO > Catalog > Products V2 > Add a product
4. In the pricing tab > Priority management  section
5. See error > The specific list is not disabled 
image
6. Select "Set a specific order for this product"
7. See error => The specific list is disabled, it shouldn't 
image
8. Click on Save
9. See error 
image

### PrestaShop version(s) where the bug happened

8.0.x

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x 
| Description?      | BO - Product V2 - Priority management form deactivation is reversed
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #28968.
| Related PRs       | 
| How to test?      | See issue
| Possible impacts? |

### CODE REVIEW

**Review by matks — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/pages/product/edit/specific-prices-manager.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default class SpecificPricesManager {
     // Enable/disabled the priority selectors depending on the priority type selected (global or custom)
     new FormFieldDisabler({
       disablingInputSelector: PriorityMap.priorityTypeCheckboxesSelector,
-      matchingValue: '0',
+      matchingValue: '1',
```
Comment (NeOMakinG) [inline admin-dev/themes/new-theme/js/pages/product/edit/specific-prices-manager.ts]: I would suggest to use a `const` or an `enum` there to make it more verbose and avoid mistakes in the future

**Review by NeOMakinG — APPROVED** [review]

**Review by jolelievre — COMMENTED** [review]
Thanks @okom3pom 

but I believe this bug has already been fixed by this PR https://github.com/PrestaShop/PrestaShop/pull/28598

I invite you to get the latest modifications from 8.0.x branch it should be fixed now, the problem came more from the component itself rather than its configuration

**Comment by ghost** [comment]
Hello @jolelievre you fix develop not 8.0 ? 

Edit : Oups i just read the description or the PR

**Comment by ghost** [comment]
Fixed by : https://github.com/PrestaShop/PrestaShop/pull/28598

**Pull request closed without merging**